api/src/main/java/org/openmrs/module/fhir/api/util/FHIRUtils.java

```diff
@@ -121,6 +121,10 @@ public static String getPractitionerStrategy() {
 		return Context.getAdministrationService().getGlobalProperty("fhir.practitioner.practitionerStrategy");
 	}
 
+	public static String getPlanDefinitionStrategy() {
+		return Context.getAdministrationService().getGlobalProperty("fhir.planDefinition.strategy");
```

Comment:
rename this to fhir.planDefinition.planDefinitionStrategy to be more consistent

Reply:
As we are planning code refactoring I believe this name should be the better solution. At this point we could start using the new convention. What do you think about it?